app/Controllers/Colaboradores/Pautas.php

```diff
@@ -66,17 +66,17 @@ public function cadastrar($idPautas = null)
 
 			$session = $this->session->get('colaboradores');
 			
-			
-			$time = new Time('-1 days');
-			$time = $time->toDateTimeString();
-			$quantidade_pautas = $pautasModel->getPautasPorUsuario($time,$session['id'])[0]['contador'];
-			if($quantidade_pautas >= $data['config']['limite_pautas_diario']) {
+
+			$time = Time::today();
+			$time = $time->toDateString();
+			$quantidade_pautas = $pautasModel->getPautasPorUsuario($time, $session['id'])[0]['contador'];
+			if ($quantidade_pautas >= $data['config']['limite_pautas_diario']) {
 				$data['erros'] = $retorno->retorno(false, 'O limite diário de pautas foi atingido. Tente novamente amanhã.', false);
 				return view('colaboradores/pautas_form', $data);
 			}
 
 			$time = new Time('-7 days');
```

Comment:
```suggestion
			$time = (new Time('-7 days'))->toDateString();
```
Desta forma vc pode excluir a linha 79.